modules/admin/admin-ui/src/main/resources/web/admin/common/js/liveedit/LiveEditModel.ts

```diff
@@ -80,13 +80,11 @@ module api.liveedit {
                 hasDefaultPageTemplate = this.pageModel.hasDefaultPageTemplate(),
                 hasApplications = this.siteModel.getApplicationKeys().length > 0;
 
-            var isRenderable = hasApplications || hasController || hasDefaultPageTemplate,
-                isRenderableSite = isSite && isRenderable,
-                isRenderableTemplate = isTemplate && hasController;
-            
-            return isRenderableSite || isRenderableTemplate || hasDefaultPageTemplate;
+            var isRenderable = hasApplications || hasController || hasDefaultPageTemplate;
+
+            return isRenderable && (isTemplate || isSite);
```

Comment:
We should also show the same "Select a controller below to set up a customized page" message above the page descriptor when creating a template as we show when creating a site.